Source/SwiftLintFramework/Rules/Lint/UnusedImportRuleExamples.swift

```diff
@@ -0,0 +1,192 @@
+struct UnusedImportRuleExamples {
+    static let nonTriggeringExamples = [
+        Example("""
+        import Dispatch // This is used
+        dispatchMain()
+        """),
+        Example("""
+        @testable import Dispatch
+        dispatchMain()
+        """),
+        Example("""
+        import Foundation
+        @objc
+        class A {}
+        """),
+        Example("""
+        import UnknownModule
+        func foo(error: Swift.Error) {}
+        """),
+        Example("""
+        import Foundation
+        import ObjectiveC
+        let 👨‍👩‍👧‍👦 = #selector(NSArray.contains(_:))
+        👨‍👩‍👧‍👦 == 👨‍👩‍👧‍👦
+        """)
+    ]
+
+    static let triggeringExamples = [
+        Example("""
+        ↓import Dispatch
+        struct A {
+          static func dispatchMain() {}
+        }
+        A.dispatchMain()
+        """),
+        Example("""
+        ↓import Foundation // This is unused
+        struct A {
+          static func dispatchMain() {}
+        }
+        A.dispatchMain()
+        ↓import Dispatch
+
+        """),
+        Example("""
+        ↓import Foundation
+        dispatchMain()
+        """),
+        Example("""
+        ↓import Foundation
+        // @objc
+        class A {}
+        """),
+        Example("""
+        ↓import Foundation
+        import UnknownModule
+        func foo(error: Swift.Error) {}
+        """),
+        Example("""
+        ↓import Foundation
+        typealias Foo = CFData
+        """, configuration: [
+            "require_explicit_imports": true
+        ])
+    ]
+
+    static let corrections = [
+        Example("""
+        ↓import Dispatch
+        struct A {
+          static func dispatchMain() {}
+        }
+        A.dispatchMain()
+        """): Example("""
```

Comment:
If you put the values (as opposed to the keys) in this dictionary on their own line, they get indented, which to my eye is easier to read.